service/src/main/java/com/codeforcommunity/processor/SiteProcessorImpl.java

```diff
@@ -108,6 +113,155 @@ public void recordStewardship(
+    siteEntriesRecord.setMultistem(addSiteRequest.isMultistem());
+    siteEntriesRecord.setCoverage(addSiteRequest.getCoverage());
+    siteEntriesRecord.setPruning(addSiteRequest.getPruning());
+    siteEntriesRecord.setCondition(addSiteRequest.getCondition());
+    siteEntriesRecord.setDiscoloring(addSiteRequest.isDiscoloring());
+    siteEntriesRecord.setLeaning(addSiteRequest.isLeaning());
+    siteEntriesRecord.setConstrictingGate(addSiteRequest.isConstrictingGate());
+    siteEntriesRecord.setWounds(addSiteRequest.isWounds());
+    siteEntriesRecord.setPooling(addSiteRequest.isPooling());
+    siteEntriesRecord.setStakesWithWires(addSiteRequest.isStakesWithWires());
+    siteEntriesRecord.setStakesWithoutWires(addSiteRequest.isStakesWithoutWires());
+    siteEntriesRecord.setLight(addSiteRequest.isLight());
+    siteEntriesRecord.setBicycle(addSiteRequest.isBicycle());
+    siteEntriesRecord.setBagEmpty(addSiteRequest.isBagEmpty());
+    siteEntriesRecord.setBagFilled(addSiteRequest.isBagFilled());
+    siteEntriesRecord.setTape(addSiteRequest.isTape());
+    siteEntriesRecord.setSuckerGrowth(addSiteRequest.isSuckerGrowth());
+    siteEntriesRecord.setSiteType(addSiteRequest.getSiteType());
+    siteEntriesRecord.setSidewalkWidth(addSiteRequest.getSidewalkWidth());
+    siteEntriesRecord.setSiteWidth(addSiteRequest.getSiteWidth());
+    siteEntriesRecord.setSiteLength(addSiteRequest.getSiteLength());
+    siteEntriesRecord.setMaterial(addSiteRequest.getMaterial());
+    siteEntriesRecord.setRaisedBed(addSiteRequest.isRaisedBed());
+    siteEntriesRecord.setFence(addSiteRequest.isFence());
+    siteEntriesRecord.setTrash(addSiteRequest.isTrash());
+    siteEntriesRecord.setWires(addSiteRequest.isWires());
+    siteEntriesRecord.setGrate(addSiteRequest.isGrate());
+    siteEntriesRecord.setStump(addSiteRequest.isStump());
+    siteEntriesRecord.setTreeNotes(addSiteRequest.getTreeNotes());
+    siteEntriesRecord.setSiteNotes(addSiteRequest.getSiteNotes());
+
+    siteEntriesRecord.store();
+  }
+
+  private List<SiteEntry> getSiteEntries(int siteId) {
+    List<SiteEntriesRecord> records =
+        db.selectFrom(SITE_ENTRIES)
+            .where(SITE_ENTRIES.SITE_ID.eq(siteId))
+            .orderBy(SITE_ENTRIES.UPDATED_AT)
+            .fetch();
+
+    List<SiteEntry> siteEntries = new ArrayList<>();
+
+    records.forEach(
+        record -> {
+          String username;
+          if (record.getUserId() == null) {
+            username =
+                db.selectFrom(ENTRY_USERNAMES)
+                    .where(ENTRY_USERNAMES.ENTRY_ID.eq(record.getId()))
+                    .fetchOne(ENTRY_USERNAMES.USERNAME);
+          } else {
+            username =
+                db.selectFrom(USERS)
+                    .where(USERS.ID.eq(record.getUserId()))
+                    .fetchOne(USERS.USERNAME);
+          }
+
+          SiteEntry siteEntry =
+              new SiteEntry(
```

Comment:
💭 I wonder how we could make this shorter, maybe a factory method on SiteEntry that takes a record and outputs a SiteEntry?

Reply:
Tried this, does not work due to Maven dependencies.